app/extensions/brave/content/scripts/flashListener.js

```diff
@@ -150,15 +152,17 @@ function insertFlashPlaceholders (elem) {
   })
 }
 
-var observer = new window.MutationObserver(function (mutations) {
-  insertFlashPlaceholders(document.documentElement)
-})
-
-if (chrome.contentSettings.flashActive != 'allow' ||
-    chrome.contentSettings.flashEnabled != 'allow') {
-  insertFlashPlaceholders(document.documentElement)
-  observer.observe(document.documentElement, {
-    childList: true,
-    subtree: true
+setTimeout(() => {
+  var observer = new window.MutationObserver(function (mutations) {
+    insertFlashPlaceholders(document.documentElement)
   })
-}
+
+  if (chrome.contentSettings.flashActive != 'allow' ||
+      chrome.contentSettings.flashEnabled != 'allow') {
+    insertFlashPlaceholders(document.documentElement)
```

Comment:
same as above, `insertFlashPlaceholders` should be called immediately on document end